### PR TITLE
#1256 fix keepBinaries for unplace

### PR DESCRIPTION
**Pull request by mximp**
#1256 fix keepBinaries

### CODE REVIEW

**Comment by mximp** [comment]
@yegor256  please check

File: eo-maven-plugin/src/test/java/org/eolang/maven/UnplaceMojoTest.java [hunk 1]
```diff
@@ -99,4 +104,59 @@ void testCleaning(@TempDir final Path temp) throws Exception {
...
+            .set(PlaceMojo.ATTR_ORIGIN, "some-keep-remove.jar")
+            .set(PlaceMojo.ATTR_HASH, new FileHash(foo));
+        new Moja<>(UnplaceMojo.class)
+            .with("placed", list.toFile())
+            .with("placedFormat", "csv")
+            .with("keepBinaries", new SetOf<>("**foo6.class"))
+            .with("removeBinaries", new SetOf<>("**foo6.class"))
+            .execute();
+        MatcherAssert.assertThat(
+            new Home().exists(foo),
```
Comment (yegor256) [inline eo-maven-plugin/src/test/java/org/eolang/maven/UnplaceMojoTest.java:154]: @mximp this is against the logic explained in [the article](https://news.eolang.org/2022-07-15-placing-and-unplacing.html). First, we should delete everything except `foo6.class`. Then, we should delete `foo6.class`. In the end, the file should be gone.

**Comment by yegor256** [comment]
@mximp see above, I think there is a mistake

File: eo-maven-plugin/src/test/java/org/eolang/maven/UnplaceMojoTest.java (hunk 1 above)
Comment (mximp) [reply eo-maven-plugin/src/test/java/org/eolang/maven/UnplaceMojoTest.java:154]: Didn't get the meaning of `removeBinaries`. Have changed the logic.

**Comment by mximp** [comment]
@yegor256 please the case now. I suppose it's in line with the article.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**